src/assets/scripts/client/airline/AirlineModel.js

```diff
@@ -68,26 +68,15 @@ export default class AirlineModel extends BaseModel {
          */
         this.flightNumberGeneration = {
             /**
-             * Length of callsign
+             * Array of callsign formats
              *
              * @memberof flightNumberGeneration
-             * @property length
-             * @type {number}
-             * @default 3
-             */
-            length: 3,
+             * @property callsignFormats
+             * @type {Array}
+             * @default ['###']
+            */
 
-            /**
-             * Whether to use alphabetical characters
-             *
-             * Used to generate a North American style registration number, minus the `N`
-             *
-             * @memberof flightNumberGeneration
-             * @property alphaNumeric
-             * @type {boolean}
-             * @default false
-             */
-            alphaNumeric: false
+            callsignFormats: ['###']
```

Comment:
please abstract this default value to a constant defined above the class or in a constants file:
```javascript
const DEFAULT_CALLSIGN_FORMAT = '###';
```

then here you can do:
```javascript
callsignFormats: [DEFAULT_CALLSIGN_FORMAT];
```